### DISCUSSION TITLE
Feature Request: Add bit <-> blob casts

### DISCUSSION THREAD

**Discussion by brendanyounger** [duckdb/duckdb#6808] [Ideas]
I have parquet files with a bit vector stored as a binary blob and I need to be able to AND the blobs together as well as test a given bit.  This would have been supported if blob -> bit conversions were allowed, but it seems that the bitvector type is being deprecated. I would appreciate that functionality being added to the blob type.

**Comment by Mytherin** [comment]
Slight misunderstanding - the bit type is not being deprecated (in fact it was added last release). The problem is that the bit <-> blob casts were never implemented, just not correctly disabled. I have updated the feature request to reflect what I think would be the preferred solution to the problem. Feel free to correct if that is not the case.
